archinstall/lib/installer.py

```diff
@@ -168,7 +168,7 @@ def set_timezone(self, zone, *args, **kwargs):
 			return True
 		else:
 			self.log(
-				f"Time zone {zone} does not exist, continuing with system default.",
```

Comment:
@dylanmtaylor Is it `Time zone` or `Timezone`? IIRC most english words are spelled apart and I think this is one of those cases?

Reply:
I have sen both used. I think Time zone may be correct, as that is what English Wikipedia uses, and they tend to know what they are doing ;)

Reply:
"There seem to be three spellings: timezone, time-zone, and time zone.

The Oxford English Dictionary defines time-zone,[^1] but curiously, also uses the time zone spelling.[^2] At the same time, timezone is the favored spelling in the US (unless talking about multiple time zones).[^3]

All three seem to be equally correct, but I would use time zone, except for a singular usage in the US for an American audience, in which case I would use timezone. Personally, I feel time-zone is a bit outdated."

https://english.stackexchange.com/a/3937